changelog/issue-1061.md

```diff
@@ -0,0 +1,5 @@
+audience: users
+level: minor
```

Comment:
https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md#levels suggests this would be `patch` since it is not "something that would motivate a user or operator to adopt the upgrade".